packages/@okta/i18n/dist/properties/login.properties

```diff
@@ -483,6 +497,8 @@ password.forgot.noFactorsEnabled = No password reset options available. Please c
 
 # Reset Password
 password.reset.title = Reset your Okta password
+password.reset.title.generic = Reset your password
```

Comment:
I like this generic/specific split, it gives us plenty of flexibility later.
Can we remove `password.reset.title` ("Reset your Okta password")?

Reply:
@nbarbettini They will be removed from i18n after we release okta-signin-widget v3.0 and pass "Okta" from okta-core. I cannot remove it now (from  i18n repo) since it what is currently used by okta-signin-widget.